File: src/P1.11.c
#include <stdio.h>

int main() {
    char nombre[30];
    printf("Introduce tu nombre: ");
    fgets(nombre, sizeof(nombre), stdin);
    printf("Hola %s Este programa esta hecho en el lenguaje de programacion: C", nombre);
    return 0;
} 

File: src/P1.11.java
import java.util.Scanner;

class Main {
  public static void main(String[] args) {
    Scanner myObj = new Scanner(System.in);
    System.out.println("Introduce tu nombre:");

    String nombre = myObj.nextLine(); 

    System.out.println("Hola " + nombre);
    System.out.println("Este programa está hecho en el lenguaje de programación: Java");
  }
}
